managed/ui/src/components/config/Storage/StorageConfiguration.js

```diff
@@ -200,6 +193,14 @@ class StorageConfiguration extends Component {
             // reset form after successful submission due to BACKUP_LOCATION value is shared across all tabs
             this.props.reset();
             this.props.fetchCustomerConfigs();
+
+            // Change to list view if form is successfully submitted.
```

Comment:
This change is in the `else` block that will handle the `create config`, but what about the `if` block handling the `update config`? I believe the same needs to be implemented there as well. Please check it out. And test it thoroughly.

Reply:
We will address this as part of follow-up PR. We will add a link for that before merging.

Reply:
Cool.